### PR TITLE
Automatically generate external id when creating an external reference (#2703)

### LINKED ISSUE #2703: Automatically generate ID in external references
## Use case

Automatically generate ID in external references

### PR DESCRIPTION
**Pull request by Archidoit**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:18-alpine AS base
+FROM node:18-alpine3.16 AS base
```
Comment (Kedae) [inline opencti-platform/Dockerfile]: You also have this on the filters PR why ?

File: opencti-platform/opencti-front/src/private/components/analysis/external_references/ExternalReferenceCreation.tsx [hunk 2]
```diff
@@ -421,12 +431,30 @@ ExternalReferenceCreationProps
...
                     />
                   )}
+                  <Field
+                    component={MarkDownField}
+                    name="description"
+                    label={t('Description')}
+                    fullWidth={true}
+                    multiline={true}
+                    rows="4"
+                    style={{ marginTop: 20, marginBottom: 20 }}
```
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/analysis/external_references/ExternalReferenceCreation.tsx:456]: Could this be a class ?

Comment (Archidoit) [reply opencti-platform/opencti-front/src/private/components/analysis/external_references/ExternalReferenceCreation.tsx:456]: Field doesn't seem to support classes= or className=  --> to be seen together

File: opencti-platform/Dockerfile (hunk 1 above)
Comment (Archidoit) [reply opencti-platform/Dockerfile]: not my changes... don't know :o

**Review by RomuDeuxfois — APPROVED** [review]

**Comment by RomuDeuxfois** [comment]
Tested ✔️

**Pull request merged**